src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -1,123 +1,61 @@
-          SkipUnchangedFiles="true"/>
-  </Target>
-
-  <!-- Executable properties -->
-  <PropertyGroup>
-    <PerfRunId Condition="'$(PerfRunId)' == ''">testResults</PerfRunId>
-    <AnalysisReportFileName>$(PerfRunId)-analysis.html</AnalysisReportFileName>
-    <TestHostExecutable Condition="'$(TargetOS)'!='Windows_NT'">./corerun</TestHostExecutable>
-    <XunitRunnerArgs>$(TargetFileName) -trait Benchmark=true -runnerhost $(TestHostExecutable) -runner $(XunitExecutable) -runid $(PerfRunId) -verbose</XunitRunnerArgs>
-    <PerfRunOutputFile Condition="'$(TargetOS)'=='Windows_NT'">%EXECUTION_DIR%\testResults.xml</PerfRunOutputFile>
-    <PerfRunOutputFile Condition="'$(TargetOS)'=='Linux'">$EXECUTION_DIR/testResults.xml</PerfRunOutputFile>
-    <XunitAnalysisReportArgs>$(PerfRunOutputFile) -html $(AnalysisReportFileName) $(AdditionalXunitAnalysisArgs)</XunitAnalysisReportArgs>
-    <XunitPerfAnalysisCsvDir Condition="'$(TargetOS)'=='Linux'">$EXECUTION_DIR/resultcsvs/</XunitPerfAnalysisCsvDir>
-    <XunitPerfAnalysisCsvDir Condition="'$(TargetOS)'=='Windows_NT'">%EXECUTION_DIR%\resultcsvs\</XunitPerfAnalysisCsvDir>
-    <XunitAnalysisCsvArgs>$(PerfRunOutputFile) -csvDir $(XunitPerfAnalysisCsvDir)</XunitAnalysisCsvArgs>
-    <AnalyzePerfResults Condition="'$(AnalyzePerfResults)' != 'false'">true</AnalyzePerfResults>
-  </PropertyGroup>
-
   <PropertyGroup Condition="'$(TargetOS)'=='Linux'">
     <PerfTestCommandDotnetExecutable>$(PerfDotNetCliDir)/dotnet</PerfTestCommandDotnetExecutable>
     <PerfAnalysisReportCommand>$(PerfTestCommandDotnetExecutable) $(PackagesDir)/$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/Microsoft.DotNet.xunit.performance.analysis.cli.dll $(XunitAnalysisReportArgs)</PerfAnalysisReportCommand>
     <PerfAnalysisCsvCommand>$(PerfTestCommandDotnetExecutable) Microsoft.DotNet.xunit.performance.analysis.cli.dll $(XunitAnalysisCsvArgs)</PerfAnalysisCsvCommand>
     <PerfTestCommand>$(PerfTestCommandDotnetExecutable) Microsoft.DotNet.xunit.performance.runner.cli.dll $(XunitRunnerArgs) $(AdditionalPerfTestArgs)</PerfTestCommand>
+	<BenchviewDir>/home/administrator/tools/Microsoft.BenchView.JSONFormat/tools</BenchviewDir>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetOS)'=='Windows_NT'">
-    <PerfTestCommandDotnetExecutable></PerfTestCommandDotnetExecutable>
+    <PerfTestCommandDotnetExecutable>PerfRunner.exe</PerfTestCommandDotnetExecutable>
     <PerfAnalysisReportCommand>$(XunitPerfAnalysisPath) $(XunitAnalysisReportArgs)</PerfAnalysisReportCommand>
     <PerfAnalysisCsvCommand>xunit.performance.analysis.exe $(XunitAnalysisCsvArgs)</PerfAnalysisCsvCommand>
-    <PerfTestCommand>$(PerfTestCommandDotnetExecutable) xunit.performance.run.exe $(XunitRunnerArgs) $(AdditionalPerfTestArgs)</PerfTestCommand>
+    <PerfTestCommand>%RUNTIME_PATH%\dotnet.exe $(PerfTestCommandDotnetExecutable)</PerfTestCommand>
+	<BenchviewDir>C:\Tools\Microsoft.BenchView.JSONFormat\tools</BenchviewDir>
```

Comment:
OK.  I can just put the nuget restore step that we do after the first time that we build the product and have the Benchview tools restored into the Tools directory.  Would that be OK?